CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 5.0.2 (unreleased)
+* Use non-deprecated method of setting prompt (see [Pry PR #1877][pry-1877])
+
+[pry-1877]: https://github.com/pry/pry/pull/1877
+
```

Comment:
Could you leave this to be done in another commit/PR please?

Reply:
No prob - updated in b54e2e6